internal/constellation/kubecmd/kubecmd_test.go

```diff
@@ -57,38 +57,6 @@ func TestUpgradeNodeImage(t *testing.T) {
 			constants.ClusterConfigurationKey: string(clusterConfBytes),
 		},
 	}
-
-	clusterConfWithKonnectivity := kubeadmv1beta3.ClusterConfiguration{
```

Comment:
Please have a look at the commit that introduced those tests and clean up everything added there.